pkg/connector/role.go

```diff
@@ -59,9 +62,11 @@ func (r *roleResourceType) Entitlements(ctx context.Context, resource *v2.Resour
 		}
 	}
 
-	users, nextPageToken, err := r.client.ListUsers(ctx, pageToken)
-	if err != nil {
-		return nil, "", nil, err
+	if len(users) == 0 {
```

Comment:
I think this could break service mode, as we'd never re-fetch the users after the first sync.

If we're going to cache stuff, we should clear the cache after every sync.

Reply:
Fixed